src/components/nav/nav-item-dropdown.js

```diff
@@ -56,7 +56,6 @@ export const BNavItemDropdown = /*#__PURE__*/ Vue.extend({
         this.extraToggleClasses, // Deprecated
         this.toggleClass,
         {
-          disabled: this.disabled,
```

Comment:
@tmorehouse Why was this disabled class removed?

Reply:
BLink adds the disabled class automatically